tests/network/vmi_multus.go

```diff
@@ -178,14 +178,13 @@ var _ = SIGDescribe("[Serial]Multus", Serial, decorators.Multus, func() {
 	})
 
 	createVMIOnNode := func(interfaces []v1.Interface, networks []v1.Network) *v1.VirtualMachineInstance {
-		vmi := libvmi.NewAlpine()
+		// Arbitrarily select one compute node in the cluster, on which it is possible to create a VMI
+		// (i.e. a schedulable node).
+		vmi := libvmi.NewAlpine(libvmi.WithNodeAffinityFor(nodes.Items[0].Name))
 		vmi.Spec.Domain.Devices.Interfaces = interfaces
 		vmi.Spec.Networks = networks
 
-		// Arbitrarily select one compute node in the cluster, on which it is possible to create a VMI
-		// (i.e. a schedulable node).
-		nodeName := nodes.Items[0].Name
-		return tests.CreateVmiOnNode(vmi, nodeName)
+		return tests.RunVMI(vmi, 90)
```

Comment:
Same, no retry is needed.

Consider this relevant to all other occurrences in this commit.